docs/old_docs/Makefile

```diff
@@ -0,0 +1,64 @@
+.PHONY: clean
```

Comment:
Why are we keeping the old Makefile?

Reply:
I moved entire directory without looking there, we will probably drop entire directory anyway

Reply:
ok lets drop it